### PR TITLE
Test plugin for WordPress 6.6.2 and 6.7-beta2

### PR DESCRIPTION
**Pull request by zaerl**
This PR updates the `Tested up to` label to WordPress 6.7-beta2.

Tested with WordPress 6.6.2 and 6.7-beta2
Tested with PHP `7.0`, `7.4`, `8.0`, `8.2`, `8.3`

How to test:
1. Clone the plugin under your working directory
2. Using `wp-env` to test, if you haven't installed it, please visit [wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#quick-tldr-instructions) for instructions
It'll be easier to have a `.wp-env.json` file in the root folder. For example, if your root folder is blogger-importer, you can clone the plugin under this folder. Create a `.wp-env.json` file; you can change the PHP version to the one you want to test with.
```json
{
  "phpVersion": "7.4",
  "plugins": ["./wordpress-importer"]
}
```
Run `wp-env start` to spin up the WordPress.

Navigate to `http://localhost:8888/wp-admin/admin.php?import=wordpress` and upload a file.
Import the content using the download attachment option and without.. it can take a while if you have a lot of content. If the script timeout, you need to `set_time_limit` to a higher value.

Check and see if the import works fine without errors and warnings.

### CODE REVIEW

File: src/readme.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ Contributors: wordpressdotorg
 Donate link: https://wordpressfoundation.org/donate/
 Tags: importer, wordpress
 Requires at least: 5.2
-Tested up to: 6.4.2
+Tested up to: 6.6.2
```
Comment (dd32) [inline src/readme.txt:6]:
```suggestion
Tested up to: 6.6
```

The Tested up to header should only contain a X.Y version number, it's not required to include the minor version.

**Review by dd32 — CHANGES_REQUESTED** [review]
WordPress 6.7 is due to be released in ~1 month, It would be ideal if we could test and bump this to that instead.

Are you able to test with the latest Beta?

**Comment by zaerl** [comment]
> Are you able to test with the latest Beta?

Good idea. There is no problem with testing with 6.7.

File: src/readme.txt (hunk 1 above)
Comment (zaerl) [reply src/readme.txt]: Tested, no problem with 6.7-beta2

**Review by dd32 — APPROVED** [review]
Works for me.

**Comment by zaerl** [comment]
@dd32 the update step [didn't pass](https://github.com/WordPress/wordpress-importer/actions/runs/11342461498/job/31542917157). Is it ok to create a `0.8.3` to incorporate the changes made [after 0.8.2](https://github.com/WordPress/wordpress-importer/commits/master/?since=2024-01-16&until=2024-10-17)?

**Comment by dd32** [comment]
#163 is the only change other than tested up to. 

If you think it's worthwhile, I guess a new minor can be released. Otherwise a plugins admin can bump the tested metadata manually.

(I can't view the logs mobile, I assume the action failed for some mundane reason)

**Comment by zaerl** [comment]
Ok, thanks. It complains about other files that have been changed and nothing more.
```
svn: warning: W155010: The node '/home/runner/svn-wordpress-importer/trunk/vendor/composer' was not found.
svn: warning: W155010: The node '/home/runner/svn-wordpress-importer/trunk/vendor/autoload.php' was not found.
?       trunk/.wordpress-org
?       trunk/LICENSE
?       trunk/README.md
!       trunk/class-wp-import.php
!       trunk/compat.php
!       trunk/parsers
!       trunk/parsers/class-wxr-parser-regex.php
!       trunk/parsers/class-wxr-parser-simplexml.php
!       trunk/parsers/class-wxr-parser-xml.php
!       trunk/parsers/class-wxr-parser.php
!       trunk/parsers.php
!       trunk/readme.txt
?       trunk/src
M       trunk/wordpress-importer.php
🛑 Other files have been modified; changes not deployed
```

**Pull request merged**